src/pages/todo/Todo.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from "react";
+import { ShowPortal } from "../../components/Components";
+import { useTodo } from "../../Context/todo-context";
+import { getTasks } from "./TodoFunc";
+import { FaPlusCircle } from "react-icons/fa";
+import { Modal, TodoCard } from "../Pages";
+import "./todo.css";
+function Todo() {
+	const [show, setShow] = useState(false);
```

Comment:
The name of the state variable can be a little more descriptive. If you are using this to show/hide a modal, you can make it showModal, setShowModal.